src/tools/genpolicy/src/registry.rs

```diff
@@ -232,7 +232,10 @@ async fn get_image_layers(
     for layer in &manifest.layers {
         if layer
             .media_type
-            .eq(manifest::IMAGE_DOCKER_LAYER_GZIP_MEDIA_TYPE)
+            .eq(manifest::IMAGE_DOCKER_LAYER_GZIP_MEDIA_TYPE) ||
```

Comment:
Please format this code using rust-analyzer. It looks like it wants to format it as:

        if layer
            .media_type
            .eq(manifest::IMAGE_DOCKER_LAYER_GZIP_MEDIA_TYPE)
            || layer.media_type.eq(manifest::IMAGE_LAYER_GZIP_MEDIA_TYPE)

Reply:
Thank you for highlighting this. I had rust-analyzer, but didn't have `formatOnSave` toggled on my vs code settings